### PR TITLE
UXG-1303: Added page guards and updated forms

### PR DESCRIPTION
**Pull request by sgreenl5**
UXG-1303 Added page leave guards and updated vies

### CODE REVIEW

File: src/vue/backend/src/mixins/pageGuard.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        routeCanLeave () {
+            return isEqual({ ...this.$store.state[this.module] }, this.localStateValues)
+        }
+    },
+    methods: {
+        formUpdate (e) {
+            const change = {
+                [e.key]: e.value
+            }
+            this.formChanges = { ...this.formChanges, ...change }
```
Comment (ralls) [inline src/vue/backend/src/mixins/pageGuard.js:22]: const declaration unnecessary?
```
this.formChanges = { ...this.formChanges, ...e }
```

File: src/vue/backend/src/mixins/pageGuard.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        },
+        saveAction () {
+            for (const item in this.formChanges) {
+                this.setItem({ key: item, value: this.formChanges[item] })
```
Comment (ralls) [inline src/vue/backend/src/mixins/pageGuard.js:26]: For consistent references to a key and some object shorthand notations - 
```
for (const key in this.formChanges) {
  this.setItem({ key, value: this.formChanges[key] })
}
```

File: src/vue/backend/src/mixins/pageGuard.js (hunk 1 above)
Comment (sgreenl5) [reply src/vue/backend/src/mixins/pageGuard.js:22]: Nope, I'll fix it

**Review by ralls — APPROVED** [review]
👍

**Review by nicmarson — APPROVED** [review]
looks good

**Pull request merged**